### DISCUSSION TITLE
Is peppy really moving away from the osu!stable version and taking more care with osu!lazer?

### DISCUSSION THREAD

**Discussion by NamOpt** [ppy/osu#25135] [Ideas] [answered]
NOTE: I'm not insulting osu!

Hello everyone, 2 months have passed since osu!stable received the August update (2023.814), and osu!lazer has received more care, having received the October update (2023.1008.1) , those things made me think that "peppy is gradually leaving osu!stable, and peppy is taking meticulous care of osu!lazer, seems like osu!stable has been ended by peppy?" 

And here are 5 reasons why osu!stable players can't switch to osu!lazer:
1. Different input feel

Osu!lazer is a rewrite of osu!stable with a new engine and UI. As a result, the input feel is different between the two clients. This can be a big adjustment for players who are used to the feel of osu!stable.

Some players have reported that osu!lazer feels more "slippery" or "floaty" than osu!stable. Others have said that the timing feels different, or that the circles are harder to hit.

While the osu!lazer team is constantly working to improve the input feel, it is still something that some players are not able to adjust to.

2. Lack of features

While osu!lazer has come a long way in recent years, it is still missing some key features that are available in osu!stable. For example, osu!lazer does not yet have support for multiplayer, replays, or custom skins.

These are all important features for many osu! players, and their absence in osu!lazer is a major reason why some players are hesitant to switch.

3. Performance

Osu!lazer is still under development, and as a result, it can be less performant than osu!stable. This can be a problem for players with lower-end computers.

The osu!lazer team is working on improving performance, but it is still something that some players need to be aware of.

4. Skin
Yes, osu!stable optimizes skins in both gameplay and GUI. This is because osu!stable uses a skinning system that is designed to be flexible and customizable. Players can use skins to change the look and feel of almost every element of the game, including the gameplay elements, the GUI, and even the cursor.

Opposite, osu!lazer is still under development, doesn't have skin support for GUI. However, the GUI skinning system in osu!lazer is not as flexible or customizable as the system in osu!stable.

5. Still regretful of memories 
It is understandable to regret the memories in osu!stable if you are switching to osu!lazer. osu!stable is a game that has been around for over a decade, and many players have made fond memories playing it.

It is natural to feel sad about leaving those memories behind, but it is important to remember that osu!lazer is a new game, and it will create new memories for you.

Overall, there are a number of reasons why osu!stable players may not be able to switch to osu!lazer. However, the osu!lazer team is constantly working to improve the client, and as it becomes more feature-complete and performant, it is likely that more and more osu!stable players will switch.

Now I just hope that peppy will return to osu!stable along with a big update for osu!stable.
                                                                                              -nam osu ne

**Comment by peppy** [comment, marked as answer]
The incorrect statements you make in this discussion don't really warrant a response.

Please actually give laser a try. All the things you mention as missing are in fact not.

As I've always said if you prefer stable you're welcome to remain there. We'll keep it running for the foreseeable future.

Check the recent video we made about how lazer will be released going forward here https://www.youtube.com/watch?v=ut_SOreClcc.

**Reply by NamOpt** [reply]
Can you release a new version for osu!stable? I see that you take care of father osu!lazer too much.

**Reply by peppy** [reply]
New version are released when there are issues to fix.

If you want to see more active development, then I think you know where to find that.
